src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -59,8 +63,29 @@ void Controller_RevitDocumentChanged(object sender, EventArgs e)
             PopulateItems();
         }
 
+        private void Updater_ElementsUpdated(object sender, ElementUpdateEventArgs e)
+        {
+            switch (e.Operation)
+            {
+                case ElementUpdateEventArgs.UpdateType.Added:
+                    break;
+                case ElementUpdateEventArgs.UpdateType.Modified:
+                    bool dynamoTransaction = e.Transactions.Contains(TransactionWrapper.TransactionName);
```

Comment:
ahh, I see you're using this to stop the loops caused by nodes, so that should be good - in other examples did you also see this check being done for added elements?

Reply:
I just found only "Selection" class has this check, https://github.com/DynamoDS/DynamoRevit/blob/433e00f19dd6cf3e3554a4317ca34dfe77e9920e/src/Libraries/RevitNodesUI/Selection.cs#L224 
And in other case, don't have checked transactions.
I investigated on Dynamo Forum, and found that some issues about infinite loop, most of them related to ElementsQueryBase nodes like All Elements of Category and All Elements of Type.